modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMessageBoards.testcase

```diff
@@ -174,4 +291,22 @@ definition {
 			workflowTask = "Update");
 	}
 
+	@description = "This is a test for LRQA-69694. Assert that the user can assign a workflow to Message Boards and verify that the process status be in Pending. "
```

Comment:
Hi Kristin! Could you, please, change the description to "_Verify that the process status is Pending when the user create a new Thread with workflow associated_"? Thank you! :)

Reply:
Updated test stub!